src/losses.jl

```diff
@@ -21,9 +21,9 @@
 #           There must be a default domain which is a Domain, which may take arguments from 
 #           the list of positional arguments. Parameters besides the scale for which there are 
 #           reasonable defaults should be included as keyword arguments (there may be none).
-#     `evaluate(l::my_loss_type, u::Float64, a::Number) ::Float64` 
+#     `evaluate(l::my_loss_type, u::Float64, a::Float64) ::Float64` 
```

Comment:
Ah, that was just for debugging something I was playing with. We can change it back.